docs/technical_reference/flowsheets/extended_BSM2.rst

```diff
@@ -56,6 +65,143 @@ Documentation for each of the property models can be found here:
+   "Primary clarifier S_F split fraction", "0.993", ":math:`\text{dimensionless}`"
+   "Primary clarifier S_I split fraction", "0.993", ":math:`\text{dimensionless}`"
+   "Primary clarifier S_N2 split fraction", "0.993", ":math:`\text{dimensionless}`"
+   "Primary clarifier S_NH4 split fraction", "0.993", ":math:`\text{dimensionless}`"
+   "Primary clarifier S_NO3 split fraction", "0.993", ":math:`\text{dimensionless}`"
+   "Primary clarifier S_O2 split fraction", "0.993", ":math:`\text{dimensionless}`"
+   "Primary clarifier S_PO4 split fraction", "0.993", ":math:`\text{dimensionless}`"
+   "Primary clarifier S_IC split fraction", "0.993", ":math:`\text{dimensionless}`"
+   "Primary clarifier S_K split fraction", "0.993", ":math:`\text{dimensionless}`"
+   "Primary clarifier S_Mg split fraction", "0.993", ":math:`\text{dimensionless}`"
+   "Primary clarifier X_AUT split fraction", "0.5192", ":math:`\text{dimensionless}`"
+   "Primary clarifier X_H split fraction", "0.5192", ":math:`\text{dimensionless}`"
+   "Primary clarifier X_I split fraction", "0.5192", ":math:`\text{dimensionless}`"
+   "Primary clarifier X_PAO split fraction", "0.5192", ":math:`\text{dimensionless}`"
+   "Primary clarifier X_PHA split fraction", "0.5192", ":math:`\text{dimensionless}`"
+   "Primary clarifier X_PP split fraction", "0.5192", ":math:`\text{dimensionless}`"
+   "Primary clarifier X_S split fraction", "0.5192", ":math:`\text{dimensionless}`"
+
+   "**Anaerobic Digester**"
+   "Anaerobic digester liquid volume", "3400", ":math:`\text{m}^3`"
+   "Anaerobic digester vapor volume", "300", ":math:`\text{m}^3`"
+   "Anaerobic digester liquid outlet temperature", "308.15", ":math:`\text{m}^3`"
+
+   "**Dewatering Unit**"
+   "Dewatering unit hydraulic retention time", "1800", ":math:`\text{s}`"
+
+   "**Thickener**"
+   "Thickener hydraulic retention time", "86400", ":math:`\text{s}`"
+   "Thickener diameter", "10", ":math:`\text{kWh/}\text{m}`"
+
+Additional Variables
+--------------------
+
+.. csv-table::
+   :header: "Description", "Symbol", "Value", "Units"
+
+   "Reactor 5 oxygen mass transfer coefficient",":math:`KLa_{R5}`", "240", ":math:`\text{hr}^{-1}`"
+   "Reactor 6 oxygen mass transfer coefficient",":math:`KLa_{R6}`", "240", ":math:`\text{hr}^{-1}`"
+   "Reactor 7 oxygen mass transfer coefficient",":math:`KLa_{R7}`", "240", ":math:`\text{hr}^{-1}`"
+   "Dissolved oxygen concentration at equilibrium",":math:`S_{O, eq}`", "8e-3", ":math:`\text{hr}^{-1}`"
+
+
+
+Additional Constraints
+----------------------
+
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "Reactor 5 mass transfer", ":math:`injection_{R5, S_{O2}} = KLa_{R5} * V_{R5} * (S_{O, eq} - S_{O, out})`"
+   "Reactor 6 mass transfer", ":math:`injection_{R6, S_{O2}} = KLa_{R6} * V_{R6} * (S_{O, eq} - S_{O, out})`"
+   "Reactor 7 mass transfer", ":math:`injection_{R7, S_{O2}} = KLa_{R7} * V_{R7} * (S_{O, eq} - S_{O, out})`"
+
```

Comment:
Still unsure whether we should include this Future refinements section. On the one hand, it let's the user know that the flowsheet isn't perfect and points out particular areas of improvement. On the other hand, we'll have to remember to update this section whenever relevant changes are applied, and I can imagine flowsheets where no future refinement is planned. The alternative would just be to track future refinements in an issue.

Let's see if @TimBartholomew or others have thoughts on this.